M/MMseqs2/build_tarballs.jl

```diff
@@ -0,0 +1,96 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MMseqs2"
+version = v"13.45111.0"
+
+# Possible build variants
+# - OpenMP (default)
+# - MPI (-DHAVE_MPI=1)
+# - single-threaded (-DREQUIRE_OPENMP=0)
+
+# TODO
+# - built-in zstd, replace with Zstd_jll? (-DUSE_SYSTEM_ZSTD=1 cmake option)
+# - os-specific build script examples under util/build_{osx,windows}
+
+# Auditor warnings
+# - freebsd (x86_64-unknown-freebsd)
+#   Warning: Linked library libomp.so could not be resolved and could not be auto-mapped
+
+# Build failures
+# - windows:
+#   - cmake can't find openmp, this check can be avoided by passing -DREQUIRE_OPENMP=0 to cmake
+#   - compile error afterwards
+#     error: ‘posix_memalign’ was not declared in this scope
+#     (and more following errors)
+# - i686: compile error due to bitwidth issues, haven't investigated more
+# - aarch64-linux with cmake option -DHAVE_ARM8=1, had to deactivate
+#   - compile wants to set -march=armv8-a+simd, but BinaryBuild
+#     wrappers don't allow that
```

Comment:
I'm not really sure why they force `-march` that way, as far as I understand that's [pretty much equivalent](https://gcc.gnu.org/onlinedocs/gcc/AArch64-Options.html) to just `-march=armv8-a` which is the base architecture anyways.  We could disable the check for `-march`, but I'd prefer simply patching the CMake file to remove the line https://github.com/soedinglab/MMseqs2/blob/efacc6904ce3654110d1878943e16ddd0e7b1280/CMakeLists.txt#L73, it's just useless